src/main/java/io/pravega/connectors/flink/sink/PravegaSinkBuilder.java

```diff
@@ -81,20 +81,20 @@ public PravegaSinkBuilder<T> forStream(final Stream stream) {
     }
 
     /**
-     * Sets the writer mode to provide at-least-once or exactly-once guarantees.
+     * Sets the delivery guarantee to provide at-least-once or exactly-once guarantees.
      *
-     * @param writerMode The writer mode of {@code BEST_EFFORT}, {@code ATLEAST_ONCE}, or {@code EXACTLY_ONCE}.
+     * @param deliveryGuarantee The delivery guarantee of {@code NONE}, {@code AT_LEAST_ONCE}, or {@code EXACTLY_ONCE}.
      * @return A builder to configure and create a sink.
      */
-    public PravegaSinkBuilder<T> withWriterMode(PravegaWriterMode writerMode) {
-        this.writerMode = writerMode;
+    public PravegaSinkBuilder<T> withDeliveryGuarantee(DeliveryGuarantee deliveryGuarantee) {
```

Comment:
We still need `withWriterMode` method to keep the compatibility.

Reply:
fixed